src/routes/docs/tutorials/apple/step-5/+page.markdoc

```diff
@@ -0,0 +1,402 @@
+---
+layout: tutorial
+title: Manage navigation 
+description: Add navigation to your SwiftUI app with Appwrite authentication.
+step: 5
+---
+
+# Router {% #router %}
+
+Add the following code to Router.swift File in the root folder
```

Comment:
I feel like we need just a little more explanation, not a lot, in each of these steps to describe what the files do and point out any documentation to swift docs that are relevant.